charts/velero/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: 1.11.0
 kubeVersion: ">=1.16.0-0"
 description: A Helm chart for velero
 name: velero
-version: 4.1.1
+version: 4.1.2
```

Comment:
Please help rebase the branch and bump the version, thank you.

Reply:
@jenting done, thanks